src/components/popover/popover.spec.ts

```diff
@@ -148,6 +148,68 @@ describe('MdlPopover', () => {
             expect(popoverComponentInstance.isVisible).toBe(false, 'main popover did not hide');
         });
     }));
+
+    it('should use user specified popover position', async(async () => {
```

Comment:
One async is enough

Reply:
@tb sorry but I can not make it work with only one async.
You see the first async wraps a test function in an asynchronous test zone.
And the second is the js language instruction for using await operator instead of promise.than chain.